apps/yapms/src/lib/components/modals/importmodal/ImportOptions.svelte

```diff
@@ -59,6 +65,58 @@
 					</label>
 				{/if}
 			</div>
+			<span class="divider m-2"></span>
+			<div class="flex gap-x-1">
+				Property Settings (Advanced)
```

Comment:
I think this should be changed to "GeoJSON Properties" or something. Just to make it a bit more clear that these are setting for GeoJSONs.